build.gradle

```diff
@@ -56,6 +56,7 @@ buildscript {
         maven {
             url "https://dl.bintray.com/cwardgar/maven/"  // For 'com.cwardgar.gretty-fork:gretty'.
         }
+        mavenCentral() // for org.jsoup

```

Comment:
I doubt very much that this line is necessary. The repositories in this file are only searched for Gradle plugins.

Reply:
To test, you can do: `./gradlew --refresh-dependencies assemble`.

Reply:
Yep, didn't need it. Need to keep the `--refresh-dependencies` flag handy for the future! I suppose intellij does that automatically when it detects a change in dependencies?